notes/stories/film/film-scifi-2020s.st.txt

```diff
@@ -791,6 +791,49 @@ murder [Zach was exposed for having deliberately electrocuted Richard Alling to
 peer pressure [Zach peer pressured some of the teens under his influence into committing wanton and callous acts of violence.]
 
 
+movie: Cosmic Sin (2021)
+========================
+
+:: Title
+Cosmic Sin
+
+:: Description
+Cosmic Sin is a 2021 American science fiction action film directed by Edward
+Drake and starring Bruce Willis and Frank Grillo.
+
+Synopsis: A disgraced former military commander is tasked back into active
+service to lead the fight against a hostile alien species that invades and
+conquers other worlds.
+
+:: Date
+2021-03-12
+
+:: References
+https://en.wikipedia.org/wiki/Cosmic_Sin
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+attack from outer space by a powerful conquering enemy [It was up to Ford and his team of elite soldiers to stop the malevolent Sigea aliens from conquering Earth.]
+extraterrestrial being [The malevolent Sigea aliens had designed on taking over the Earth.]
+weapons of mass destruction [After much build up, a "Q-bomb" super-weapon used to destroy the Sigea's attack armada and home star system, saving Earth from being conquered.]
+genocide [Ford had perpetrated a genocide against the Zafdie rebels. The ethics of using the "Q-bomb" super-weapon against the potentially hostile Sigea aliens was discussed. The story culminated in the act of genocide that was the humans using a "Q-bomb" to destroy the Sigea aliens' home star system.]
+
+:: Minor Themes
+colonization of Mars [The viewer learns that the first colony on Mars was founded in the year 2031, but had somehow catastrophically failed in 2281.]
+space colonization [The Alliance had a colony, named Ellora, on some far-flung planet.]
+mechanical man [A humanoid robot bartender struggled to pour Ford a shot of what appeared to be whisky.]
+atrocities of war [The viewer learns that Ford, under Alliance orders, dropped a "Q-bomb" on a rebel colony, killing 70 million people.]
+duty to disobey illegal orders [Ford stated that he was just following Alliance orders in response to being called out for using a weapon of mass destruction, which resulted in the deaths of 70 million people.]
+space portal [Ford and his party traveled to Ellora via a "Quantum Jump Gate" to stop the Sigea aliens.]
+hologram [A holographic country music singer was playing on stage at the bar.] {49:00, 1:24:00}
```

Comment:
Don't really care, but I thought you had decided not to leave {}'s?

Reply:
Whoops. I missed taking those {} out. They've now been eliminated.